cases/util/getTomlFile.js

```diff
@@ -2,7 +2,7 @@ import { fetch } from "./fetchShim";
 import TOML from "toml";
 
 export default async function(domain) {
-  let response = await fetch(domain + "/.well-known/stellar.toml");
+  let response = await fetch(domain + ".well-known/stellar.toml");
   const text = await response.text();
   const toml = TOML.parse(text);
```

Comment:
Lets ensure here that WEB_AUTH_ENDPOINT and TRANSFER_SERVER have their traling slashes instead of doing it in each test

Reply:
done!